EasyPost.Tests/BatchTest.cs

```diff
@@ -174,11 +179,12 @@ public async Task TestLabel()
             Batch batch = await CreateOneCallBuyBatch();
 
             // Uncomment the following line if you need to re-record the cassette
-            // Thread.Sleep(2000); // Wait enough time for the batch to process buying the shipment
+            // Thread.Sleep(10000); // Wait enough time to process
```

Comment:
Let's remove this first occurrence of the sleep timer. There is no need to sleep between creation and buying.